diff -dur -x '*.orig' gevent-1.0.2.orig/known_failures.py gevent-1.0.2/known_failures.py
--- gevent-1.0.2.orig/known_failures.py	2015-05-23 14:04:13.000000000 +0200
+++ gevent-1.0.2/known_failures.py	2015-07-14 15:20:53.000000000 +0200
@@ -16,6 +16,89 @@
     # Sometimes fails with AssertionError: ...\nIOError: close() called during concurrent operation on the same file object.\n'
     # Sometimes it contains "\nUnhandled exception in thread started by \nsys.excepthook is missing\nlost sys.stderr\n"
     "FLAKY test__subprocess_interrupted.py",
+
+######################################################
+# Problems which should be investigated
+
+# AttributeError: 'module' object has no attribute '_url_collapse_path_split'
+'monkey_test --Event test_httpservers.py',
+'monkey_test test_httpservers.py',
+
+#   AssertionError: The 304 response must have no body
+# and:
+#       File "/home/users/jajcus/rpm/BUILD/gevent-1.0/gevent/hub.py", line 331, in s
+#         return greenlet.switch(self)
+#     Timeout: 1 second: test timed out
+'FLAKY test__pywsgi.py',
+
+######################################################
+# Tests which fail when building with system libev
+
+# test__core_stat.py fails on 32-bit builders when using system
+# libev
+#
+##  Watcher <stat at 0x86b078c> reacted after 0.0019 seconds (write)
+##  Watcher <stat at 0x86b078c> reacted after 0.0019 seconds (unlink)
+##  Traceback (most recent call last):
+##    File "test__core_stat.py", line 54, in <module>
+##      assert watcher.attr is None, watcher.attr
+##  AssertionError: posix.stat_result(st_mode=2001, st_ino=0L, st_dev=581509237251768320L, st_nlink=1000, st_uid=0, st_gid=0, st_size=5983298792366014472L, st_atime=1393095309, st_mtime=0, st_ctime=-1)
+#* * * -u test__core_stat.py
+
+
+# When  building with system libev the 'fileno()' method of the
+# loop object is not implemented. It seems to be used only be the
+# test code, though.
+
+# | /usr/bin/python -u test__destroy.py
+#   Traceback (most recent call last):
+#     File "test__destroy.py", line 14, in <module>
+#       assert saved_loop.fileno() is None, saved_loop
+#   AttributeError: 'gevent.core.loop' object has no attribute 'fileno'
+'FLAKY test__destroy.py',
+
+# | /usr/bin/python -u test_close_backend_fd.py
+#   Traceback (most recent call last):
+#     File "test_close_backend_fd.py", line 11, in <module>
+#       fileno = hub.loop.fileno()
+#   AttributeError: 'gevent.core.loop' object has no attribute 'fileno'
+'FLAKY test_close_backend_fd.py',
+
+######################################################
+# Tests which may fail on our builders
+
+# attempts to bind to port 53
+'FLAKY test_ares_timeout.py',
+
+# depends on external service
+'FLAKY test__socket_ssl.py',
+
+# depends on external URL accessibility
+'FLAKY monkey_test test_urllib2net.py',
+'FLAKY monkey_test --Event test_urllib2net.py',
+
+# requires webpy, not in PLD
+# has expectations on local TCP ports availability
+'FLAKY test___example_servers.py',
+
+#    File "test__socket.py", line 140, in test_sendall_timeout
+#      self.assertRaises(self.TIMEOUT_ERROR, client.sendall, data_sent)
+#  AssertionError: timeout not raised
+'FLAKY test__socket.py',
+
+# uses DNS and external URLs,
+# requires 'sendfile' python module
+'FLAKY test__examples.py',
+
+# expects local TCP port 6000 being available
+'FLAKY test__example_echoserver.py',
+
+# another network test
+'FLAKY test__example_portforwarder.py',
+
+# DNS
+'FLAKY test__socket_dns.py',
+
 ]
 
 if sys.version_info[:2] <= (2, 6):
