--- gevent-1.2.1/src/greentest/test__doctests.py.orig	2017-01-12 14:15:25.000000000 +0100
+++ gevent-1.2.1/src/greentest/test__doctests.py	2017-03-14 22:18:48.660258016 +0100
@@ -68,9 +68,12 @@
             # 'cannot access'
             (re.compile('cannot access non_existent_file: No such file or directory'),
              'non_existent_file: No such file or directory'),
+            (re.compile("cannot access 'non_existent_file': No such file or directory"),
+             'non_existent_file: No such file or directory'),
             # Python 3 bytes add a "b".
             (re.compile(r'b(".*?")'), r"\1"),
             (re.compile(r"b('.*?')"), r"\1"),
+            (re.compile(r'^"(.*?)"$'), r"'\1'"),
         ))
 
         tests_count = 0

